loader

```diff
@@ -144,7 +144,10 @@ def load(invocid, size, output_method, report_method, dist='gaussian', stddev=DE
     payloadgen = gen_methods[payload_gen]
     sep = ' - '
     sep_len = len(sep)
-    prefix = "loader seq - %s - " % invocid
+    if log_format == 'string':
+      prefix = "loader seq - %s - " % invocid
+    else:
+      prefix = "{\"message\":\"loader seq - %s - " % invocid
```

Comment:
`prefix = '{"message":"loader seq - %s - ' % invocid`

Reply:
Without \", does loader generate a log like this?  (well, let me test it... :)
{"log":"{\"message\":\"loader seq - a0bb3e8de2f140d9af3ed249371b2ef2 - 0000030458 - cde8h5ggqb.....ru196en365k\"}\n","stream":"stdout","time":"2018-07-23T20:12:20.747417064Z"}

Reply:
it should, yes
```
>python
Python 2.7.15 (default, May 16 2018, 17:50:09) 
[GCC 8.1.1 20180502 (Red Hat 8.1.1-1)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> prefix = '{"message":"loader seq - %s - ' % 'invocid'
>>> print prefix
{"message":"loader seq - invocid - 
```

Reply:
Yes, it does!  I love python. :)  Fixing...